internal/app/server/server.go

```diff
@@ -12,10 +12,12 @@ import (
 	"google.golang.org/grpc"
 )
 
+const defaultLogLevel = "info" // TODO make configurable
```

Comment:
can't this be an env variable?

Reply:
Plan is to have it set via a global config, I want to create a separate PR that'll handle config management soon.